include/ola/messaging/StringMessageBuilder.h

```diff
@@ -38,7 +38,7 @@ class StringMessageBuilder: public FieldDescriptorVisitor {
     explicit StringMessageBuilder(const std::vector<std::string> &input)
         : m_input(input) {
     }
-    ~StringMessageBuilderVisitor() {}
+    ~StringMessageBuilder() {}
```

Comment:
Xcode.  I don't recall seeing any warnings when compiling the library...  just when using the header in an Xcode project.